### ISSUE TITLE
Tidy up public S3 bucket

### ISSUE THREAD

**Issue by rufuspollock** [cybergreen-net/pm#107]
I suspect this data is now obsolete: http://bits.cybergreen.net/dev/clean/

Also, the reference files don't seem to be in http://bits.cybergreen.net/dev/stats/ (should they be)?

**Comment by zelima** [comment]
@rufuspollock correct, we should remove `dev/clean` as well as `prod/clean`  that are the leftovers from old runs, before DW refactor. 
Reference files should not be anywhere on s3 any  more they are placed in Github:
- https://github.com/cybergreen-net/refdata-asn
- https://github.com/cybergreen-net/refdata-risk
- https://github.com/cybergreen-net/refdata-country

**Comment by rufuspollock** [comment]
@zelima sounds good. Re reference data files i think we need to still link them from the download page. In a perfect world we'd make a data package and list them ;-)

**Comment by aaronkaplan** [comment]
+1 good idea
